funfact/lang/_ast.py

```diff
@@ -73,6 +73,10 @@ def ein(
     ):
         '''pairwise einsum-like operations between tensors'''
 
+    @primitive(precedence=9)
+    def tran(src: _ASNode, idst: _ASNode):
```

Comment:
It is meant as `destimation indices` :sweat_smile: I started with `dst_indices` and then changed to `idst` since the first one is too verbose. Now that you mentioned this, I think **given that the context is clear, we can just call it `indices`'.**